pkg/controller/gittrackobject/handler.go

```diff
@@ -176,13 +176,14 @@ func (r *ReconcileGitTrackObject) handleDefaultUpdateStrategy(gto farosv1alpha1.
 
 	// Update was successful
 	r.sendEvent(gto, corev1.EventTypeNormal, "UpdateSuccessful", "Successfully updated child %s %s/%s", child.GetKind(), child.GetNamespace(), child.GetName())
-	log.Printf("Updated child %s %s/%s\n", child.GetKind(), child.GetNamespace(), child.GetName())
+	r.log.V(0).Info("Child Updated")
```

Comment:
Shouldn't this be "Child updated" (note lowercase u) as the log line above reads "Child created".